corehq/apps/app_manager/models.py

```diff
@@ -393,6 +402,25 @@ class LoadUpdateAction(AdvancedAction):
     auto_select = SchemaProperty(AutoSelectCase, default=None)
     show_product_stock = BooleanProperty(default=False)
     product_program = StringProperty()
+    case_index = SchemaProperty(CaseIndex)
```

Comment:
Not sure if you want to do this or not but if you add `default=None` then JsonObject won't create an object if there isn't one.